float64 velAY
float64 velBX
float64 voltM
int32 adrs
int32 pote
bool lan

